### ISSUE TITLE
the program killed for no reason, with specific apk file

### ISSUE THREAD

**Issue by Gunkkk** [MobSF/Mobile-Security-Framework-MobSF#1582]
## ENVIRONMENT

```
OS and Version: windows10
Python Version: 3.7
MobSF Version: matser-branch 2020/11/4
```

## EXPLANATION OF THE ISSUE
it happens when analysize the specific app， it seems  something wrong with the binary analysis.

```
What happens, under which versions, under what conditions, when, and what were you expecting instead.
```

## STEPS TO REPRODUCE THE ISSUE

```
1. This is the first step
upload the specific app to do the static analysis, the mobsf will be killed for no reason
2. This is the second step
restart the mobsf ,Traceback (most recent call last):                                                                                                                                 'security_score'] = score(context['code_analysis'])                                                                                                         
TypeError: 'NoneType' object is not subscriptable                                                                                                              
 [ERROR] 07/Nov/2020 21:42:48 - 'NoneType' object is not subscriptable                                                                                           [ERROR] 07/Nov/2020 21:42:48 - Internal Server Error: /StaticAnalyzer/        
3. Further steps, etc.

```

## LOG FILE
first time:
when info
Saving to Database.
it killed
second time:
[ERROR] 07/Nov/2020 21:42:47 - Fetching from DB                                                                                                                 Traceback (most recent call last):                                                                                                                   
mobsf\Mobile-Security-Framework-MobSF-master\StaticAnalyzer\views\android\db_interaction.py", line 54, in get_context_from_db_entry                                                                                                                                                                 'binary_analysis': python_list(db_entry[0].BINARY_ANALYSIS),                                                                                                  mobsf\Mobile-Security-Framework-MobSF-master\MobSF\utils.py", line 229, in python_list                                              
return ast.literal_eval(value)                                                                                                                                
File "C:\ProgramData\Anaconda3\lib\ast.py", line 46, in literal_eval                                                                                              node_or_string = parse(node_or_string, mode='eval')                                                                                                          
 File "C:\ProgramData\Anaconda3\lib\ast.py", line 35, in parse                                                                                                     return compile(source, filename, mode, PyCF_ONLY_AST)                                                                                                         
File "<unknown>", line 1                                                                                                                                          
[{'name': 'lib/armeabi-v7a/libandromeda.so', 'nx': {'is_nx': True, 'severity': 'info', 'description': 'The shared object has NX bit set. This marks a memory page non-executable making attacker injected shellcode non-executable.'},
```
Paste the contents of logs/debug.log here or attach the log file.
```

**Comment by ajinabraham** [comment]
Please share the APK that causes this crash.

**Comment by Gunkkk** [comment]
> Please share the APK that causes this crash.

https://1drv.ms/u/s!Ao62m3bO5pGNmBB2zxD84dQPokuj?e=opnw6C

**Comment by ajinabraham** [comment]
Reproduced the issue

**Comment by ajinabraham** [comment]
This should be fixed in 3.2.0
